packages/type-definitions/src/types_10720.ts

```diff
@@ -14,8 +14,8 @@ export const types10720: RegistryTypes = {
     identifier: 'AccountId32',
     accounts: 'Vec<DidApiAccountId>',
     w3n: 'Option<Text>',
-    serviceEndpoints: 'Vec<RawServiceEndpoints>',
-    details: 'RawDidDetails',
+    serviceEndpoints: 'Vec<DidServiceEndpointsDidEndpoint>',
+    details: 'DidDidDetails',
   },
   RawServiceEndpoints: {
```

Comment:
@ntn-x2 I wasn't sure whether I can remove these as well because they are still relevant for the RPCs. Maybe I should adjust the types in a new `types_10750.ts` file and remove all `Raw*` stuff in there?

Reply:
We don't have RPC definitions anyway. So it's fine to remove also the types used for those.

Reply:
People willing to use the RPC stuff will most likely have to use an older version of the type definitions, which still contains both RPCs and their type definitions.